### PR TITLE
Register view

### PR DESCRIPTION
**Pull request by SilviaPabon**
- Navigates from signup to login and vice versa
- Checks if there is a valid mail, if username is a valid string and a valid password
- Shows an alert if user was successfully created and navigates to login view and also if something were wrong
- Makes the request to api /signup

### CODE REVIEW

**Review by PedroChaparro — APPROVED** [review]
Works, gw 🎉

**Comment by andres123dbh** [comment]
imagen

It does not recognize the password 12345678Aa@

**Comment by PedroChaparro** [comment]
@SilviaPabon, try with this regexp, it's more complete: 

`/^(?=.*[A-Z])(?=.*[a-z])(?=.*\d)(?=.*[\)\(\;\.\,\]\[\}\{\_=@$!¡#%*?¿&^+-/<>|~])[A-Za-z\d\)\(\;\.\,\]\[\}\{\_=@$!¡#%*?¿&^+-/<>|~]{8,}$/`

File: src/pages/Signup.tsx [hunk 1]
```diff
@@ -1,6 +1,163 @@
...
+    onSubmit: async (values) => {
+      const [response, error] = await signupRequest(
+        values.email,
+        values.username,
+        values.password
+      );
+      if (error && response?.message) {
+        showErrorToast(response?.message);
+      } else {
+        showInfoToast('User created succesfully, now confirm your Email');
```
Comment (PedroChaparro) [inline src/pages/Signup.tsx:48]: @SilviaPabon I think it's better to show a success alert instead of an information one.

**Comment by SilviaPabon** [comment]
@andres123dbh and @PedroChaparro the changes requested were made! Thanks!

**Review by andres123dbh — APPROVED** [review]

**Pull request merged**